src/lang/artifact.test.ts

```diff
@@ -313,6 +316,7 @@ sk2 = startSketchOn(XY)
           ],
           sketch: {
             id: expect.any(String),
+            isClosed: false,
```

Comment:
I would've expected any Solid's sketch to be closed. You can't extrude an open path.

Reply:
Oh, that's a great catch. I think the problem is that it's actually fine to run extrude without a close, if the path happens to be naturally closing (e.g. three lines that loop back on each other to form a triangle). I wish we could make that an error, but oh well, too late because that would be a breaking change. Maybe for kcl2.

In the meantime, I'll make sure that extruding or revolving a sketch closes it.